reana_client/api/client.py

```diff
@@ -125,6 +150,14 @@ def create_workflow(self, user, organization, reana_spec):
                     "{status_code}".format(
                         status_code=http_response.status_code))
 
+        except HTTPError as e:
+            logging.debug(
+                'Analysis status could not be retrieved: '
```

Comment:
Should it be 'Analysis could not be created'?

Reply:
updated!